DiceKeys/Views/Backup/Stickeys/TransferDie.swift

```diff
@@ -31,12 +31,17 @@ struct TransferDieInstructions: View {
 
     var body: some View {
         VStack(alignment: .leading) {
-            Text("Find the \(face.letter.rawValue) die.").font(.title)
+            Text("Find the \(face.letter.rawValue) die.")
+                .font(.title)
+                .minimumScaleFactor(0.5)
```

Comment:
Does .minimumScaleFactor do anything without a lineLimit?

(Asking because I don't know and want to know if it helps without that.)

Reply:
Yes, it will. Because it scale down the text of it, if spacing is less.